### PR TITLE
scalability framework: allow configuring regression threshold

### PR DESCRIPTION
**Pull request by nrainer-materialize**

### CODE REVIEW

File: test/scalability/mzcompose.py [hunk 1]
```diff
@@ -49,7 +49,7 @@
     Postgres(),
 ]
 
-REGRESSION_THRESHOLD_AS_PERCENT_DECIMAL = 0.2
+DEFAULT_REGRESSION_THRESHOLD_AS_PERCENT_DECIMAL = 0.2
```
Comment (nrainer-materialize) [inline test/scalability/mzcompose.py:52]: Giving you also the chance to challenge the name again 😀

Comment (def-) [reply test/scalability/mzcompose.py:52]: The name still makes no sense to me since nothing is seen as a "part of every hundred"

**Review by def- — APPROVED** [review]

File: test/scalability/mzcompose.py (hunk 1 above)
Comment (nrainer-materialize) [reply test/scalability/mzcompose.py:52]: Any alternative suggestions?

Comment (nrainer-materialize) [reply test/scalability/mzcompose.py:52]: I don't think it is clearer but I will adjust it.

**Pull request merged**